libopenage/pyinterface/functional.h

```diff
@@ -63,7 +63,7 @@ class Func {
 
 	// for construction from lambdas and other callables (from C++).
 	template<typename F>
-	Func(F &&f) {
+	Func(F &&f) noexcept {
 		this->fptr = f;
```

Comment:
This will copy the function object and may actually throw. Note that strictly speaking this is not a move constructor.